repairnator/src/main/java/fr/inria/spirals/repairnator/process/step/AbstractStep.java

```diff
@@ -94,7 +94,7 @@ public void setState(ProjectState state) {
         }
     }
 
-    protected Logger getLogger() {
+    public Logger getLogger() {
```

Comment:
This logger should be only available in the current package (subclasses) not in other classes, so you shouldn't change the visibility here.